### PR TITLE
Resolves #181 : Make a better CONTRIBUTING.md

### LINKED ISSUE #181: [Doc] Better contributing.md
make a better [contributing.md](https://github.com/mdl29/scratchy/blob/main/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by yannis-mlgrn**
Resolves #181

### CODE REVIEW

**Review by Draeggon — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md
Comment (yannis-mlgrn) [reply CONTRIBUTING.md:30]: yes i will add this : "you can create a issue and in them explain your problem. Finally after resolve him you can push your pull request"

File: README.md
Comment (Draeggon) [inline README.md:29]: add a ">" before the emoji
but why did you used that emoji ?

File: CONTRIBUTING.md
Comment (Draeggon) [inline CONTRIBUTING.md:31]: make an whole section dedicated to create an issue like for creating a pr

File: CONTRIBUTING.md
Comment (Draeggon) [inline CONTRIBUTING.md:44]: explain keyword usage of "fixes",  "resolves" and the other

File: CONTRIBUTING.md
Comment (Draeggon) [inline CONTRIBUTING.md:47]: the code owner will  squash and merge

**Review by Draeggon — CHANGES_REQUESTED** [review]
and for example_issue.png it is an image of a pr not an issue

File: CONTRIBUTING.md
Comment (Benvii) [inline CONTRIBUTING.md:3]: Titles are in bold by defaut, I think it's useless to use `**...**` in titles

File: CONTRIBUTING.md
Comment (Benvii) [inline CONTRIBUTING.md:5]: Maybe add link to the issue page  ? simply add a `/issues` add the end of the link

File: CONTRIBUTING.md
Comment (Benvii) [inline CONTRIBUTING.md:22]: add something like `if it's work in progress, set the [pull request status to draft](https://github.blog/2019-02-14-introducing-draft-pull-requests/)`

File: README.md
Comment (Benvii) [reply README.md:29]: The emoji seem adapted for me.

File: CONTRIBUTING.md
Comment (Benvii) [reply CONTRIBUTING.md:31]: Approve, make a section about "Creating issue", that will also explain each label meanings

**Review by Benvii — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md
Comment (yannis-mlgrn) [reply CONTRIBUTING.md:3]: I mean it's more beautiful haha

Comment (Benvii) [reply CONTRIBUTING.md:18]: It doesn't change anything except adding characters in the source file ^^

File: CONTRIBUTING.md
Comment (Benvii) [inline CONTRIBUTING.md:76]: `wait for a reviewer`

**Review by Benvii — CHANGES_REQUESTED** [review]
Just saw one error.

**Comment by yannis-mlgrn** [comment]
> Just saw one error.

Where? 😂

**Review by Benvii — DISMISSED** [review]
Good for me 👍

**Comment by Benvii** [comment]
I can't merge it until @Draeggon has not validated it

File: CONTRIBUTING.md
Comment (Draeggon) [inline CONTRIBUTING.md:71]: it is a screenshot of a pr not an issue

**Review by Draeggon — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md
Comment (yannis-mlgrn) [reply CONTRIBUTING.md:59]: i forgot to remove this line

File: CONTRIBUTING.md
Comment (yannis-mlgrn) [reply CONTRIBUTING.md:67]: because when I resolve a review you add 7 other new review 😂

Comment (Draeggon) [reply CONTRIBUTING.md:67]: code need to be clean just take the time :joy:

Comment (yannis-mlgrn) [reply CONTRIBUTING.md:67]: solves and fixes have the same meaning, right?

File: CONTRIBUTING.md
Comment (yannis-mlgrn) [reply CONTRIBUTING.md:69]: I think is good to explain again if a beginner want to contribute

File: CONTRIBUTING.md
Comment (Draeggon) [reply CONTRIBUTING.md:67]: almost but you need to provide a link to doc that explain how to link issues using these words

Comment (yannis-mlgrn) [reply CONTRIBUTING.md:67]: ohhh you don't want merge that i think is right no ?
keep this changement for contributiong.md V2 haha

Comment (Draeggon) [reply CONTRIBUTING.md:67]: dude it is just a file we won't do versionning of it

Comment (Draeggon) [reply CONTRIBUTING.md:67]: but yeahh i want the repo to be clean for V1 even some things like dockerfile aren't there

Comment (yannis-mlgrn) [reply CONTRIBUTING.md:67]: ok I fought a good (tutoriel](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue), you think is right ?

Comment (Draeggon) [reply CONTRIBUTING.md:67]: maybe put this link:
https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
he has an anchor directly to the section that we care about

Comment (yannis-mlgrn) [reply CONTRIBUTING.md:67]: hmm okay thanks

File: CONTRIBUTING.md
Comment (Draeggon) [reply CONTRIBUTING.md:69]: in that case for consistency you should do it for the issue as well

File: CONTRIBUTING.md
Comment (yannis-mlgrn) [reply CONTRIBUTING.md]: whichone ?

Comment (Draeggon) [reply CONTRIBUTING.md]: when your pull request link to an issue

File: CONTRIBUTING.md
Comment (yannis-mlgrn) [reply CONTRIBUTING.md]: Yes yes

Comment (Draeggon) [reply CONTRIBUTING.md]: what is the word france want to now

File: CONTRIBUTING.md
Comment (yannis-mlgrn) [reply CONTRIBUTING.md]: hmm no have typos

Comment (Draeggon) [reply CONTRIBUTING.md]: ??

File: CONTRIBUTING.md
Comment (yannis-mlgrn) [reply CONTRIBUTING.md]: I must put `KEYWORD #ISSUEID : title of the issue`
KEYWORD : fixes, Resolves. See [here](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

Comment (Draeggon) [reply CONTRIBUTING.md]: Capture d’écran du 2021-05-27 22-55-44
description not pr name

File: CONTRIBUTING.md
Comment (yannis-mlgrn) [reply CONTRIBUTING.md]: wtf I go sleep I don't have the strength to understand 😂

Comment (Draeggon) [reply CONTRIBUTING.md]: just replace "when your pull request link to an issue" by "when your pull request is ready link to an issue"

File: CONTRIBUTING.md
Comment (yannis-mlgrn) [reply CONTRIBUTING.md]: what ?

Comment (Draeggon) [reply CONTRIBUTING.md]: replace `KEYWORD #ISSUEID : title of the issue` by `ISSUEID : title of the issue` it will work believe me

Comment (yannis-mlgrn) [reply CONTRIBUTING.md]: no, I rhink is better to keep `KEYWORD #ISSUEID : title of the issue`
like `Resolves #91: add scratchy service`

Comment (Draeggon) [reply CONTRIBUTING.md]: why ? github automatically give the name `ISSUEID title of the issue` when you open the pr

**Review by landerneRPi — APPROVED** [review]

**Pull request merged**
